src/components/App.js

```diff
@@ -1,21 +1,40 @@
 /* eslint-disable no-unused-vars */
 import './App.css';
+import React, { useState } from 'react';
 import Header from './Header';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 import calculate from '../logic/calculate';
 
-function App() {
+const App = () => {
```

Comment:
Our aim is to convert this functional component to a class component, initialize the state in the constructor, implement handleClick in the constructor.......Kindly make this change